lib/modules/manager/helmfile/utils.ts

```diff
@@ -23,3 +31,41 @@ export async function localChartHasKustomizationsYaml(
     upath.join(helmfileYamlParentDir, release.chart, 'kustomization.yaml')
   );
 }
+
+export function parseDoc(packageFileContent: string): Doc {
+  const doc = yaml.load(packageFileContent);
+  return DocSchema.parse(doc);
+}
+
+export function getRepositories(doc: Doc): Repository[] {
+  if (is.nullOrUndefined(doc.repositories)) {
+    return [] as Repository[];
+  }
+
+  return doc.repositories;
+}
+
+export function isOCIRegistry(repository: Repository): boolean {
+  if (is.nullOrUndefined(repository.oci)) {
+    return false;
+  }
+
+  return repository.oci;
```

Comment:
```suggestion
  return repository.oci === true;
```
isn't this much simpler? so we can inline this ?